### DISCUSSION TITLE
Welcome to spam!

### DISCUSSION THREAD

**Discussion by JC5** [community/community#27247] [Discussions] [answered]
Already the community discussions are flooded with spam, misplaced posts, and other nonsense. For the time being, this is a hard pass for me.

Could you add a "Report" button?

**Comment by sawyerbivens** [comment]
100% agree with a report button, not sure if Github has a moderation team in place for reviewing reports though

**Comment by jfullstackdev** [comment]
just want to comment too, I think they will be implementing it, the old Forum can report spam and nonsense

but there is already this feature

https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-organization

they may take action anytime

if you started an Org focused solely on Discussions, you can do this too

**Reply by JC5** [reply]
It's not about my organization, it's about these community forums.

**Reply by jfullstackdev** [reply]
yes, as I've mentioned, they may take action anytime the same practice in the Old forum, they still have the control of this Forum, 

you can also report users by reporting them, going directly to their profile

**Reply by JC5** [reply]
Yes, they *may*. I'm urging them to actually do it. Without feedback like this, I'm not confident it will actually happen.

**Comment by jricciardi** [comment, marked as answer]
Hi @JC5! Thank you for the feedback — we are very much aware of the spam (and empathize! we don't want spam either :) ). We have some automated spam tools in the process of being refined to minimize the disruption both here and in GitHub Discussions all across GitHub. We don't currently have a 'report' button available, but I'm happy to pass along your support for that feature.

**Reply by jricciardi** [reply]
FYI I'm moving this post from the Welcome Center over to the Discussions category & adding some relevant tags. The idea of a "report" button is feedback that is widely applicable to GitHub Discussions as a product, so this will have more visibility there 😄

**Reply by sayhiben** [reply]
For what it's worth, I'd also appreciate a "report" button in addition to any automated moderation

**Reply by pingu8007** [reply]
Hey I just been flooded and came here and is amazed you guys do nothing in last 5 months to fight against user generated junk!
So, what are you waiting for? Waiting for a childporn video to ring your fire alarm? I guess you guys better doing something now before angry users poking around your DMCA report or hostmaster mailbox or whatever :)

**Reply by JC5** [reply]
All the spam I got is already gone.

**Comment by airtower-luna** [comment]
This is critical. I tried reporting some spammers using the existing rather cumbersome "report user" feature yesterday, and in three out of four cases the reply was a form letter about the options repository maintainers have to moderate discussions. Well, reaching maintainers (GitHub staff) was exactly what I was trying to do. :confused: 

It's fair enough if the "Trust & Safety" team isn't in charge of moderation here, but in that case we need an easy way to report to the actual moderators. And at least for spam/vandalism there usually doesn't need to be a complicated report, most of it is pretty obvious once a human sees it. Moderators' decisions on which reported posts are or aren't spam could even be used to improve those automated systems. For problems other than spam (harassment, inappropriate remarks, etc.) it can be more complex.

And a report button would be useful for discussions in other repositories, too: Not every maintainer has a publicly visible mail address, or people observing spam etc. might not want to reveal theirs, and people might not want to open an issue/discussion to report where the spammer (or harasser, or whatever, it's applicable to other undesirable behavior, too) can see.

**Comment by hollabaq86** [comment]
Hi there! We are seeing an uptick in spam Issues being created in our repos, and the steps to block and report the abuse are incredibly cumbersome.

We need a more streamlined way to block and report spammy content, or perhaps enable more strict controls on submitting issues like captcha/2FA? This is a growing issue for us.

**Comment by dslatkin** [comment]
@jricciardi Not sure if it was happening this way before, but my first time being pinged by a spam bot. 🙃 Unsure about the safety of those links, but the maintainer of this project already took it down (thankfully we have a Discord in this instance).

<img width="926" alt="screenshot of spam message" src="https://user-images.githubusercontent.com/9263640/209888838-90aa2ab4-e0d4-4cf5-a145-509f38f86271.png">

**Reply by DSDmark** [reply]
Same things happening to me

**Reply by lperron** [reply]
same here. Amazingly, I get it by mail, but it is not there on the github site.

**Reply by HarelM** [reply]
I have the same issue in my repo, this started to happen yesterday night:
![image](https://user-images.githubusercontent.com/3269297/209929928-2b2608d6-7b51-446e-bf94-b00ceb27843a.png)

The posts are removed fast (probably automatic) and I think the users are also being removed.
But the users are being created faster than being removed, and I still get a mail to my inbox and this spams my inbox.

HELP!!!!

**Reply by gandham-santhosh** [reply]
Same happening to us. We could not find an audit log related to this comment creation and deletion as well. Appreciate it if someone from Github acknowledge it as spam and deleted it through GitHub spam detection tools. As a precautionary measure, we have set an interaction limit so that users that have recently created their account will be unable to interact with this organization's repositories.

**Comment by vpereira** [comment]
I received a spam message right now, via discussions, and there is no way to report it. Link to the specific spam https://github.com/AndreCerqueira/HorseManager2022/discussions/7#discussion-4700814

**Reply by DannyBen** [reply]
For me, this is the first time I have seen spam on GitHub, and I can also confirm like the last few posts in this thread, there seems to be a new "way" for spammers to reach us. It seems like there is a good logic in GitHub's spam detection to delete these posts, but it would appear that the mechanism that informs users **by mail** about these posts is trigger happy, and sending the spam by mail.

I am mentioning it, since I much rather have this spam not reach me, than having to "report" it - and it seems like GitHub spam detection is almost getting it right.

**Reply by HarelM** [reply]
+1

**Comment by cytech** [comment]
+1
6 different emails
![image](https://user-images.githubusercontent.com/7809634/209962181-2bb3231b-5c1d-427d-988e-4bae106f7039.png)

**Comment by geckolinux** [comment]
I've received at least 10 different notifications during the past 24 hours from porn spammers on my Github project's issues. Github needs to implement some sort of rate restriction and privilege restriction on new Github user accounts, this can't continue. Also multiple reports on the web:

- https://www.reddit.com/r/github/comments/zxq399/spammers_also_now_leveraging_github/
- https://news.ycombinator.com/item?id=34167965

**Reply by DannyBen** [reply]
If it is a repo under your control, you can easily limit interactions, as suggested [in this comment](https://github.com/orgs/community/discussions/27247#discussioncomment-4548139).

In your repo's Settings > Moderation Options > Interaction Limits:

![image](https://user-images.githubusercontent.com/2405099/209964518-c9394025-38f2-4cbb-a3dc-3a2e2da85ebf.png)

**Reply by geckolinux** [reply]
Thank you, very helpful. The option for "Users that have recently created their account will be unable to interact with the repository." is what I need.

**Comment by Akash1134** [comment]
Hi there, GitHub is responding to concerns expressed by community members regarding recent spam posts & comments. Thanks for sharing and we appreciate your patience.

**Comment by matthew-dean** [comment]
The Less.js repo discussions are now being [flooded with Spam](https://github.com/less/less.js/discussions) (I've since disabled the Discussions tab, because it is now useless). However, blocking users doesn't remove all of their posts! And they are spread between multiple users. How can this be managed?

**Comment by mehulrajdev** [comment]
What is going on?

**Comment by devtmj1123** [comment]
I am not a spammer , but my account was being marked as spam , so I try to remove my primary and backup mail and it didn't work.

**Comment by aurelienheude** [comment]
this is a test answer
